src/main/java/org/example/homework_nr_8/Animal.java

```diff
@@ -0,0 +1,21 @@
+package org.example.homework_nr_8;
+
+public interface Animal {
+    String name = null;
```

Comment:
All variables in interfaces must be of following name convention: String NAME_OR_SOMETHING_ELSE. It's because any variable you declare in an interface are final by default